go.mod

```diff
@@ -51,6 +51,12 @@ require (
 	k8s.io/client-go v0.26.7
 )
 
+require (
```

Comment:
This new block in here probably isn't necessary.

Reply:
Let me fix this.